server/tests/io/feed_parsers/belga_newsml_1_2_test.py

```diff
@@ -392,3 +392,36 @@ def test_audio_ingest(self):
         self.assertIn('href', item['renditions']['original'])
         self.assertIn('media', item['renditions']['original'])
         self.assertEqual(item['renditions']['original']['mimetype'], 'audio/mpeg')
+
+
+class BelgaNewsMLOneAudioIngestTestCase2(TestCase):
+    filename = 'belga_newsml_1_2_audio2.xml'
+
+    def setUp(self):
+        super().setUp()
+        dirname = os.path.dirname(os.path.realpath(__file__))
+        fixture = os.path.normpath(os.path.join(dirname, '../fixtures', self.filename))
+        provider = {'name': 'test', 'config': {'path': os.path.join(dirname, '../fixtures')}}
+        parser = BelgaNewsMLOneFeedParser()
+
+        def get_file_side_effect(value):
+            media_path = os.path.normpath(os.path.join(dirname, '../fixtures', value))
+            with open(media_path, 'rb') as f:
+                return BytesIO(f.read())
+
+        parser._get_file = MagicMock(side_effect=get_file_side_effect)
+
+        with open(fixture, 'rb') as f:
+            self.xml_root = etree.parse(f).getroot()
+            self.item = parser.parse(self.xml_root, provider)
+
+    def test_audio_ingest(self):
+
```

Comment:
can you pls assert len on `self.item` length so it's obvious how many items are parsed from the file?

Reply:
Sure